style-sheets/primevue-components-overrides.scss

```diff
@@ -271,5 +295,57 @@ label {
 }
 
 /* ------------------------------------ primevue card--------------------------------- */
+.#{variables.$primevue-prefix}-card {
+    box-shadow: none;
+    border: 1px solid colors.$gray-30;
+    padding: 32px;
+}
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-body {
+    padding: 32px;
+}
+
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-content {
+    padding: 0;
+    margin: 0;
+}
+
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-title {
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+    margin-bottom: 0;
+}
 
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-subtitle {
+    color: var(--light-theme-text-text-secondary, #606062);
+    font-size: 14px;
+    line-height: 18px;
+    letter-spacing: 0.16px;
+}
+
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-footer {
+    padding: 0;
+}
 /* ------------------------------------ primevue table --------------------------------- */
+.#{variables.$primevue-prefix}-datatable .#{variables.$primevue-prefix}-datatable-thead > tr > th {
+    /* Light Theme/Layer accent/$layer-accent-hover-01 */
+    background: #d2d2d4;
+    height: 32px;
+    padding: 0 1rem;
+    font-size: 14px;
+    font-weight: 700;
+    letter-spacing: 0.1599999964237213px;
+    text-align: left;
+    border: none;
+}
+.#{variables.$primevue-prefix}-datatable .#{variables.$primevue-prefix}-datatable-tbody > tr > td {
+    text-align: left;
+    border: 1px solid #dee2e6;
```

Comment:
check if this color exists in the design tokens

Reply:
Thanks, I have updated it.